Cargo.toml

```diff
@@ -50,17 +47,17 @@ rust_1_30 = ["snafu-derive/rust_1_30", "doc-comment"]
 exclude = [
   "compatibility-tests/compile-fail",
   "compatibility-tests/v1_34",
-  "compatibility-tests/v1_30",
+  "compatibility-tests/v1_31",
   "compatibility-tests/futures-0.1",
   "compatibility-tests/futures",
   "compatibility-tests/without-backtrace",
 ]
 
 [dependencies]
 snafu-derive = { path = "snafu-derive", version = "0.5.0" }
+doc-comment = "0.3.1"
```

Comment:
Let's keep `default-features = false`. This helps minimize compile time where possible.

Reply:
Sorry about that; just pushed the fix in the 1.30->1.31 commit.